doc/source/quickstart.rst

```diff
@@ -246,6 +260,26 @@ output format and to provide the best looking image. For more information
 on saving datasets and customizing enhancements see the documentation on
 :doc:`writers`.
 
+
+Slicing and subsetting scenes
+=============================
+
+Array slicing can be done at the scene level in order to get excerpts with consistent navigation throughout. Note that this does not take into account scenes that may include multi-resolution channels.
+
+  >>> scene_slice = global_scene[2000:2004, 2000:2004]
+  >>> vis006_slice = scene_slice['VIS006']
+  >>> vis006_slice_meas = vis006_slice.values
+  >>> vis006_slice_lon, vis006_slice_lat = vis006_slice.attrs['area'].get_lonlats()
+
+To subset multi-resolution data, use the `.crop` function. 
+
+  >>> scene_llbox = global_scene.crop(ll_bbox=(-4.0, -3.9, 3.9, 4.0))
```

Comment:
Should it be made explicit that pixel indices don't make sense in the multiresolution case ? and that this is a lon/lat box ?

Reply:
Took a swing at clarifying this - let me know if it's not a net improvement.